server/ldap_check.rb

```diff
@@ -86,6 +86,8 @@ def is_user_in_admin_hash user
       @updateCount += 1
       @lastUpdate = Time.now()
       save_group_members configuration['group']
+      # Might get a group with no members or that doesn't exist.
+      @admin_hash = Hash.new unless(@admin_hash)
```

Comment:
I see admin_hash is tested here, but I don't see where it's being set prior to this.  Does save_group_members do that?  If so, I would think that it's the responsibility of that function to make sure admin_hash has a hash, an empty one at least.